src/main/java/exercise_5/TestShape.java

```diff
@@ -2,6 +2,7 @@
 
 public class TestShape {
     public static void main(String[] args) {
+
         Shape s1 = new Circle("red", false, 5.5);
```

Comment:
s1 is not a good name, maybe redCircle?

Reply:
Yes, you are correct. Code readability is very important. I'll fix it